test/functional/apps/home/_navigation.js

```diff
@@ -30,8 +29,18 @@ export default function ({ getService, getPageObjects }) {
   const fromTime = '2015-09-19 06:31:44.000';
   const toTime = '2015-09-23 18:31:44.000';
 
-  describe('Kibana browser back navigation should work', function describeIndexTests() {
+  const getBasePath = () => {
+    if (process.env.TEST_KIBANA_URL) {
+      const myURL = new URL(process.env.TEST_KIBANA_URL);
+      return `${myURL.protocol}//${myURL.host}`;
+    }
+    const protocol = process.env.TEST_KIBANA_PROTOCOL || 'http';
+    const host = process.env.TEST_KIBANA_HOST || 'localhost';
```

Comment:
This should be TEST_KIBANA_HOSTNAME

Reply:
yes, but also TEST_KIBANA_HOST is in use, take a look:
https://github.com/elastic/kibana/blob/08de89b588502be8fc3e3ced71149dd6f813a12a/Jenkinsfile#L113
what I did was: 
1) check TEST_KIBANA_HOSTNAME
2) if not available check TEST_KIBANA_HOST
3) if not available fall back to localhost